app/assets/javascripts/social.js

```diff
@@ -0,0 +1,19 @@
+$(window).load(function() {
+
+  ///// Social media buttons
+  if ($('.shareSave').length) {
+    (function(d, s, id) {
+      var js, fjs = d.getElementsByTagName(s)[0];
+      if (d.getElementById(id)) return;
+      js = d.createElement(s); js.id = id;
+      js.src = "//connect.facebook.net/en_US/all.js#xfbml=1";
+      fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));
+
+    (function() {
+      var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true;
+      po.src = 'https://apis.google.com/js/plusone.js';
```

Comment:
If the `js.src` declaration above on line 9 assigns a scheme-relative URL, could this do the same? (Removing `https:`)